### DISCUSSION TITLE
Starting HLS stream

### DISCUSSION THREAD

**Discussion by Btrem69** [bluenviron/mediamtx#456] [Questions]
Been looking and can't find this in the discussions.

Been using jQuery to pop open a small window with the stream:

$("#videoMe").click(function () {
            let vidWindow = window.open("http://localhost:8888/proxied", "video", vidParams,)
            vidWindow.moveTo(500, 100);
            vidWindow.focus();
        })

but when I try to embed it in my web app with a video tag, the server doesn't get notified to start streaming.  I've tried:

paths:
  proxied:
    # url of the source stream, in the format rtsp://user:pass@host:port/path
    source: rtsp://admin:xxxxxxxxxx@192.168.xx.xxx:554/cam/realmonitor?channel=1?subtype=0
    sourceOnDemend: no

and 

  # command to run when a client connects to the server.
  # this is terminated with SIGINT when a client disconnects from the server.
  # the server port is available in the RTSP_PORT variable.
  runOnConnect: yes
  # the restart parameter allows to restart the command if it exits suddenly.
  runOnConnectRestart: yes

Thanks for you help.

**Comment by aler9** [comment]
HLS streams can't be used directly as source of a \<video> tag, this is not supported by all browsers except Safari. You have to load the stream with a library like https://github.com/video-dev/hls.js/

[a comment by an automated account is not shown]
